docs/_tutorials/automatic-tensor-parallelism.md

```diff
@@ -126,6 +126,8 @@ The following model families have been successfully tested with automatic tensor
 - bigbird_pegasus
 - bloom
 - camembert
+- chatglm2
```

Comment:
Thanks @Yejing-Lai, we will get this merged after the 2 PRs that add the support for these models.